Sources/Model/TransferMethod/HyperwalletBankAccountQueryParam.swift

```diff
@@ -23,16 +23,6 @@ public class HyperwalletBankAccountQueryParam: HyperwalletTransferMethodQueryPar
     private enum QueryParam: String {
         case type
     }
-    /// The bank account type.
-    public var type: QueryType?
-
-    /// Represents the Bank Account types.
-    public enum QueryType: String {
-        /// The Bank account type
-        case bankAccount = "BANK_ACCOUNT"
-        /// The Wire account type
-        case wireAccount = "WIRE_ACCOUNT"
```

Comment:
How are you going too restrict that only BANK_ACCOUNT and  WIRE_ACCOUNT  is set for this query param?